playlist.py

```diff
@@ -0,0 +1,91 @@
+from PyQt5.QtWidgets import QDialog, QFileDialog, QAbstractItemView
+from playlist_ui import Ui_Dialog
+from clip import load_song_from_file, verify_ext
+import json
+from os.path import basename, splitext
+
+
+def getSongs(file_names):
+    r = []
+    for f in file_names:
+        try:
+            r.append(load_song_from_file(f))
```

Comment:
I tested playlist, it works perfectly, if users have big songs that not fit in memory they just don't use playlist.